schemas/interface.yml

```diff
@@ -0,0 +1,31 @@
+- name: interface
+  title: Interface
+  group: 2
+  short: Fields to describe observer interface information.
+  description: >
+    The interface fields describe information about ingress and egress interfaces reported 
+    by an observer.
+
+  reusable:
+    top_level: false
+    expected:
+      - observer.ingress
+      - observer.egress
+
+  type: group
+  fields:
+
+    - name: id
+      level: core
+      type: keyword
+      example: 1
+      description: >
+        Interface ID as reported by the system (typically SNMP interface ID).
+
+    - name: name
+      level: extended
+      type: keyword
+      example: GigabitEthernet0/1, eth0, outside
+      description: >
+        Optional interface name as reported by the system.  Could be short (eth0), long (gigabitethernet0/2), 
```

Comment:
I'm all for flexibility, but I wonder if users will want to track more than of those at the same time? E.g. the computer-generated one ("eth0") as well as the customized one ("dmz").

Reply:
typically I'ver seen an observer use 1 or the other, but that begs the question of mixing interface name (gigabitethernet0/1) with logical name (outside)

Maybe we should add alias in the first go round? (for logical interface custom names in e.g. firewalls)

Reply:
I added alias - we can pick another name, but I think we would be better off NOT mixing actual interface names (eth0, gigabitethernet0/1, etc) with aliases (outside, inside, dmz, etc)

Reply:
also - dropped everything to extended